server/backend/api/app/crud/crud_application.py

```diff
@@ -23,7 +24,7 @@ def get_application(db: Session, application_id: int):
 
 def get_application_role_assignments(
     db: Session, application_id: int, requester: schemas.Requester
-) -> List[models.FamUserRoleXref]:
+) -> List[schemas.FamApplicationUserRoleAssignmentGet]:
```

Comment:
Nothing change, only change return type from 'model' to 'schema' (still the same type return at router)